paper/paper.md

```diff
@@ -0,0 +1,83 @@
+
+While different other agent-based models have been developed for the study of energy markets, such as PowerACE [@bublitzAgentbasedSimulationGerman2014] and AMIRIS [@schimeczekAMIRISAgentbasedMarket2023], the possible integration of reinforcement learning methods into the behavioral strategies of market agents is currently unique to ASSUME.
+This feature allows for the exploration of new market designs and dynamics in energy markets using a common open-source simulation framework.
+Further unique features of `ASSUME` are the extensive market abstraction which allows to define complex multi market scenarios as shown in [@maurerMarketAbstractionEnergy2023].
+ASSUME is designed to provide results which are easily comparable and interactable.
+This is set in contrast to other agent-based simulation frameworks, which are often designed to be used for a specific use case.
+Instead, various moving parts can be configured, such as the market abstraction, the agent properties and bidding strategies, or the reinforcement learning methods.
+
+# Architecture
+
+`ASSUME` builds on the open-source agent-based simulation library [mango-agents](https://pypi.org/project/mango-agents/) to model the simulation world and to give users continuous feedback during the simulation runs.
+Interaction with the results of the simulation is possible through the Grafana dashboards, which do have direct access to the simulation database.
+Additionally, writing the output to CSV files is supported for scenarios in which a database is not available.
+
+New scenarios can be created by providing CSV files, accessing simulation data from a database or by using the object-oriented API, which makes it possible to integrate new classes, maket mechanisms and bidding strategies as well.
+The core overview of available classes and the interaction with the world is shown in \autoref{fig:architecture}.
+
+![Basic Architecture of the ASSUME simulation\label{fig:architecture}](../docs/source/img/architecture.svg)
+
+# Publications
+
+ASSUME has been used to investigate the usage of complex order types like block bids in wholesale markets [@adamsBlockOrdersMatter2024] as well as for the integration of demand-side flexibility and redispatch markets in [@khanraEconomicEvaluationElectricity2024].
+Studies of applicability of reinforcement learning methods in energy markets were tackled in [@harderHowSatisfactoryCan2024], while an analysis of explainable AI methods was appliedd to ASSUME in [@miskiwExplainableDeepReinforcement2024].
+For better interoperability with other energy system model data, adapters to interact from ASSUME with PyPSA and AMIRIS are available and make a comparison to other renomated market simulation tools possible [@maurerKnowYourTools2024].
+
+# Acknowledgements
+
+Kim K. Miskiw, Nick Harder and Manish Khanra thank the German Federal Ministry for Economic Affairs and Climate Action for the funding of the ASSUME project under grant number BMWK 03EI1052A.
+
+# References
```

Comment:
Assume allows modelling of any potential energy market or not? The data we provide is focused on Germany. Hence I would not state as clearly that it is made for primarly Germany